### PR TITLE
[do not merge] Skip internal underflow

### LINKED ISSUE #96: xt_not_rote should be xt_not_rot
I just noticed this in a recent PR.  Not sure where the e came from, but the assembly name should be the same as the text in the word (with the symbols spelled out as words).  I wouldn't worry about this in the PR (unless it's super easy) as it can be cleaned up later.

### PR DESCRIPTION
**Pull request by patricksurry**
Here's work in progress that adds `w_` labels.  Also:

- some improvements for `#`; 
- simplifies return stack operator compilation (removes strip-table checks with new ST flag); 
- moves and simplifies headers.asm with def_nt macro; 
- shortens `s_quote` by moving lookup table outside word;
- adds words/README.md;
- fixes #96

### CODE REVIEW

File: words/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+This is where all of Tali's native forth words are defined.
+
+At the top level `taliform.asm` includes `all.asm` here which defines
```
Comment (SamCoVT) [inline words/README.md]: I think you wanted taliforth.asm here

**Comment by SamCoVT** [comment]
I think this looks good so far.  You're welcome to include results.txt (I'll clean it up if it makes a mess when merging) as it may show improvement just in the words that already have cycle tests (+/- the normal cycle test deviation due to addresses crossing pages).

File: words/headers.asm [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+.if "block" in TALI_OPTIONAL_WORDS
+#def_nt buffstatus
+#def_nt buffblocknum
+#def_nt blkbuffer
+#def_nt scr,            NN
+#def_nt blk,            NN
+;TODO these mention HC but don't have the flag?  or is the HC wrong in block_read* ?
+#def_nt block_write,    UF+NN, "block-write"    ; Deferred words need the HC (Code Field) flag.
+#def_nt block_write_vector, NN, "block-write-vector" ; Deferred words need the HC (Code Field) flag.
+#def_nt block_read,     UF+HC+NN, "block-read"  ; Deferred words need the HC (Code Field) flag.
```
Comment (patricksurry) [inline words/headers.asm:341]: The block_read and block_read_vector words include the HC flag, and the block_write and block_write_vector mention the flag but don't add it.  Should they all have it?  Or is it mistaken and none of them should have it (that's 
 my guess but haven't explored that code much)?

**Comment by SamCoVT** [comment]
I think they shouldn't have the code field flag.  I was trying to do them as deferred words, but it's really difficult to do that in ROM.  I believe the current solution is that these are all helper words that use the actual vector held in RAM in the system variables (not sure if it's in the ZP variables or the RAM variables). I suspect this is a documentation error (eg. the comment is wrong) for the way things are currently done.

**Comment by patricksurry** [comment]
i'm going to split this up into smaller parts to make it easier to review

**Comment by patricksurry** [comment]
Split into #106 #107 and #108

I'll add one more with just the w_ label changes and then close this

**Comment by patricksurry** [comment]
#109 now has just the original xt_ to w_ changes, without any header macro-izing.  I'll explore that separately.

**Pull request closed without merging**